.github/workflows/xcube_publish_pypi.yml

```
@@ -0,0 +1,37 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: publish to PyPI
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+    name: Publish to PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install build
+    - name: Build package
+      run: |
+        export PYPI_XCUBE_NAME="xcube-core"
```

```suggestion
        export XCUBE_PYPI_NAME="xcube-core"
```

Sorry for being picky here, but usually we prefix xcube-related environment variables by "XCUBE_" (sometimes "XC_") rather than prefixing them with the system context.